networking/multiple_networks/configuring-additional-network.adoc

```diff
@@ -92,13 +92,15 @@ spec:
     type: Raw
 ----
 <1> An array of one or more additional network configurations.
-
 <2> The name for the additional network attachment that you are
 creating. The name must be unique within the specified `namespace`.
-
 <3> The namespace to create the network attachment in. If
-you do not specify a value, then the `default` namespace is used.
-
+you do not specify a value then the `default` namespace is used.
```

Comment:
This is out of scope to your work and up to you, but while you're making edits to this file anyways, consider removing the hard wrapping in these lines (i.e. making sure that the sentences on lines 95-96 and 97-98 are each on their own lines instead of being split in the middle)

Reply:
:heavy_check_mark: